src/Xamarin.CodeAnalysis/RequiresSuperAttributeCodeFixProvider.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@kzu Apache License?

Reply:
resolved this comment because it's a duplicate of another comment.